### PR TITLE
docs: add docs for the `parse_influxdb` vrl function

### PR DESCRIPTION
**Pull request by jorgehermo9**
Relates to https://github.com/vectordotdev/vrl/pull/982

### CODE REVIEW

File: website/cue/reference/remap/functions/parse_influxdb.cue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	]
+	internal_failure_reasons: [
+		"`value` is not a valid InfluxDB line protocol string.",
+	]
+	return: types: ["object"]
+
+	examples: [
+		{
+			title: "Parse InfluxDB line protocol"
+			source: #"""
```
Comment (jorgehermo9) [inline website/cue/reference/remap/functions/parse_influxdb.cue:35]: This example is the same from the vrl repo PR. 

Is is better to have a single example depicting all possible field value types (like this one), or should I split it in 3 or so,
so this example is not "bloated"?

File: website/cue/reference/remap/functions/parse_influxdb.cue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		Parses the `value` as an [InfluxDB line protocol](https://docs.influxdata.com/influxdb/v1/write_protocols/line_protocol_reference/)
+		string, producing a list of Vector-compatible metrics".
+		"""
+	notices: [
+		"""
+			The only metric type that is produced is a `gauge`. Each metric name is prefixed with the `measurement` field, followed
+			by an underscore (`_`), and then the `field key` field.
+			""",
+	    """
+			`string` are the type that is not supported as a field value,
```
Comment (jorgehermo9) [inline website/cue/reference/remap/functions/parse_influxdb.cue:15]:
```suggestion
			`string` is the only type that is not supported as a field value,
```

**Review by michaelcretzman — APPROVED** [review]

File: website/cue/reference/remap/functions/parse_influxdb.cue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		{
+			name:        "value"
+			description: "The string representation of the InfluxDB line protocol to parse."
+			required:    true
+			type:        ["string"]
+		},
+	]
+	internal_failure_reasons: [
+		"`value` is not a valid InfluxDB line protocol string.",
+		"field set contains a field value of type `string`.",
```
Comment (jorgehermo9) [inline website/cue/reference/remap/functions/parse_influxdb.cue:33]: Added two more failure reasons, related to https://github.com/vectordotdev/vrl/blob/b0daea0245de03ecae01622417f17093c8736594/src/stdlib/parse_influxdb.rs#L77

File: website/cue/reference/remap/functions/parse_influxdb.cue (hunk 1 above)
Comment (jszwedko) [reply website/cue/reference/remap/functions/parse_influxdb.cue:41]: I think this example is ok and shows all of the possible field values well.

File: website/cue/reference/remap/functions/parse_influxdb.cue [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+remap: functions: parse_influxdb: {
+	category: "Parse"
+	description: """
+		Parses the `value` as an [InfluxDB line protocol](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/)
+		string, producing a list of Vector-compatible metrics".
+		"""
+	notices: [
+	   """
+		    This function will return a log event with the shape of a Vector-compatible metric, but not a metric event itself.
```
Comment (jszwedko) [inline website/cue/reference/remap/functions/parse_influxdb.cue:11]: 👍 thanks for including this note.

**Review by jszwedko — APPROVED** [review]
Thanks @jorgehermo9 ! This looks good to me. I'll merge it once VRL is upgraded in Vector.

**Pull request merged**